helper/config.go

```diff
@@ -93,6 +95,8 @@ const (
 
 	DefaultTendermintNode = "tcp://localhost:26657"
 
+	DefaultMainnetSeeds string = "f4f605d60b8ffaaf15240564e58a81103510631c@159.203.9.164:26656,4fb1bc820088764a564d4f66bba1963d47d82329@44.232.55.71:26656,2eadba4be3ce47ac8db0a3538cb923b57b41c927@35.199.4.13:26656,3b23b20017a6f348d329c102ddc0088f0a10a444@35.221.13.28:26656,25f5f65a09c56e9f1d2d90618aa70cd358aa68da@35.230.116.151:26656"
```

Comment:
Could we include `DefaultTestnetSeeds`?

Reply:
We can, but in description of the task (in notes) there is this sentence: "Only use predefined seed nodes for mainnet since our mumbai testnet is only available to private partners.". If necessary, we can put a default value for Mumbai Testnet as well, but somebody has to provide me the default value.

Reply:
AFAIK there's default seed nodes defined in the docs https://docs.polygon.technology/docs/integrate/full-node-deployment/#setup-full-node-for-testnetv4mumbai-testnet

Those should be correct

Reply:
default seeds for testnet is pushed (also unit test)